.github/actions/certify-openshift-images/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM alpine:latest
+
+COPY entrypoint.sh /home/entrypoint.sh
+RUN chmod +x /home/entrypoint.sh
+# Code file to execute when the docker container starts up (`entrypoint.sh`)
+ENTRYPOINT ["/home/entrypoint.sh"]
```

Comment:
A few files have "no new line" thing, probably better to fix it 😅

Reply:
This symbol is not shown here but it is present in the [file changes](https://github.com/mongodb/mongodb-atlas-kubernetes/pull/646/files).